lib/preface_specs/free_selective.mli

```diff
@@ -18,6 +18,40 @@
 
 (** {1 Structure anatomy} *)
 
+(** The natural transformation for [Free Selective] to [Selective]. *)
+module type TO_SELECTIVE = sig
+  type 'a t
+  (** The type held by the [Free Selective]. *)
+
+  type 'a f
+  (** The type held by the {!module:Preface_specs.Functor}. *)
+
+  type 'a selective
+  (** The type held by the [Selective]. *)
+
+  type natural_transformation = { transform : 'a. 'a f -> 'a selective }
+
+  val run : natural_transformation -> 'a t -> 'a selective
+  (** Run the natural transformation over the [Free selective]. *)
+end
+
+(** The natural transformation for [Free Selective] to [Monoid]. *)
+module type TO_MONOID = sig
```

Comment:
The type seems exactly the same of `Free_applicative.TO_MONOID` why not using it?

Reply:
For documentation purpose.